apps/storybook.namekit.io/stories/Nameguard/RatingIcon.stories.tsx

```diff
@@ -5,52 +5,68 @@ import { RatingIcon, RatingIconSize } from "@namehash/nameguard-react";
 const meta: Meta<typeof RatingIcon> = {
   component: RatingIcon,
   title: "Nameguard/RatingIcon",
-  parameters: {
-    controls: {
-      exclude: ["size", "isInteractive"],
-    },
-  },
   argTypes: {
     rating: {
-      options: [Rating.pass, Rating.warn, Rating.alert],
+      options: Object.keys(Rating),
+      control: { type: "select" },
+    },
+    size: {
+      options: Object.keys(RatingIconSize),
       control: { type: "select" },
     },
+    isInteractive: { control: { type: "boolean" } },
   },
 };
 
 export default meta;
 
 type Story = StoryObj<typeof RatingIcon>;
 
+export const Pass: Story = {
+  args: {
+    rating: Rating.pass,
+    size: RatingIconSize.medium,
+  },
+};
+
+export const Warn: Story = {
+  args: {
+    rating: Rating.warn,
+    size: RatingIconSize.medium,
+  },
+};
+
+export const Alert: Story = {
+  args: {
+    rating: Rating.alert,
+    size: RatingIconSize.medium,
+  },
+};
 export const Large: Story = {
   args: {
     rating: Rating.pass,
     size: RatingIconSize.large,
   },
```

Comment:
Hey @lightwalker-eth 

1. Yes, my bad for not removing the files, I just did it!
2. Not needed since only /apps folder has changed

Set boolean: done ✅